src/CommonGLPI.php

```diff
@@ -138,6 +138,21 @@ public static function getType()
         return get_called_class();
     }
 
+    /**
+     * Used by generic forms for details and logging purposes
+     *
+     * @return string
+     */
+    public static function getFormServiceName(): string
+    {
+        return '';
+    }
```

Comment:
As this will probably only be used for logs, `log` should appear in the name. Also, it will probably be used everytime an event is logged for this class, not only in the generic forms.

```suggestion
    /**
     * Default service identifier to use when logging an event attached to this class.
     *
     * @return string
     */
    public static function getDefaultLogService(): string
    {
        return '';
    }
```